src/components/PartnersContent/PartnersTile/PartnersTile.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FunctionComponent } from "react";
+import { Button, ButtonSize } from "@govtechsg/tradetrust-ui-components";
+import { PartnersTileProps } from "../types";
+
+export const PartnersTile: FunctionComponent<PartnersTileProps> = ({ data }) => {
+  const { companyName, description, logo, websiteLink } = data;
+  return (
+    <div className="max-w-[380px] min-w-[380px] bg-white rounded-xl shadow-md p-5 mb-10 mr-8">
+      <p className="text-2xl">{companyName}</p>
```

Comment:
👀

Reply:
?

Reply:
dont like the max-w min-w?

Reply:
it's redundant, why not `w-[380px]`? or even better is to refer to news page on using traditional flex + w fractional combi

Reply:
ok got it

Reply:
this width makiing it dynamic ok, but i was making it exact to design so we can safe to say all tiles will look the same in all screen sizes. but i would think now its a little dynamic, for bigger screens its a bit bloated thou.

Reply:
ic, can you replace `w-[380px]` with `w-auto` and apply `-mx-4` to the parent flex?

Reply:
i think its fine now, i let jerry see the design of the tiles and he is ok with it.

Reply:
![Screenshot 2023-04-25 at 9 27 46 AM](https://user-images.githubusercontent.com/4774314/234152559-e238356a-a303-4c25-8b6e-27b55751a5a5.png)

Reply:
ok, updated